### PR TITLE
ExtendedPicker: Allow edit selected items

### PR DESCRIPTION
**Pull request by amyngu**
#### Pull request checklist

- [x] Include a change request file using `$ npm run change`

#### Description of changes
- Add 'Edit' context menu item
- Allow selected item to be edited, by accepting floating picker render/props
   - In ExtendedPicker case, will reuse the floating picker given in ExtendedBasePicker props
- Expose a "Force resolve suggestion" method on FloatingPicker

### CODE REVIEW

File: packages/experiments/src/components/SelectedItemsList/SelectedPeopleList/Items/EditingItem.tsx [hunk 1]
```diff
@@ -0,0 +1,120 @@
...
+export interface IEditingSelectedPeopleItemProps extends ISelectedPeopleItemProps {
+  // tslint:disable-next-line:no-any
+  onEditingComplete: (oldItem: any, newItem: any) => void;
+  onRenderFloatingPicker?: (props: IBaseFloatingPickerProps<IExtendedPersonaProps>) => JSX.Element;
+  floatingPickerProps?: IBaseFloatingPickerProps<IExtendedPersonaProps>;
+  getEditingItemText?: (item: IExtendedPersonaProps) => string;
+}
+
+export class EditingItem extends BaseComponent<IEditingSelectedPeopleItemProps, IPeoplePickerItemState> {
+  private editingInput: HTMLInputElement;
```
Comment (dzearing) [inline packages/experiments/src/components/SelectedItemsList/SelectedPeopleList/Items/EditingItem.tsx]: privates are underscored.

**Comment by amyngu** [comment]
@dzearing not sure why screener is now failing. The only changes I made between my last green screener was change privates to prefix with underscores.

**Comment by betrue-final-final** [comment]
There was an issue with screener where it didn't or something else in the system didn't see a breaking change that got into Master. We're working on fixing that now.

**Comment by amyngu** [comment]
@betrue-final-final is there a way to restart my screener test?

**Comment by joschect** [comment]
@amyngu The easiest way to do this is close the request, then reopen it. I've done that for you so it should start up again.

**Comment by amyngu** [comment]
@joschect @dzearing any other comments :) thanks!

**Pull request merged**